app/src/main/java/com/cornellappdev/scoop/ui/components/post/MapRideCard.kt

```diff
@@ -0,0 +1,312 @@
+                        .size(29.dp)
+                        .align(CenterVertically),
+                    contentDescription = stringResource(R.string.calendar_icon_description)
+                )
+
+                rideState.value.dateOfTrip?.let { date ->
+                    Text(
+                        text = date,
+                        modifier = Modifier.align(CenterVertically),
+                        style = MaterialTheme.typography.body1
+                    )
+                }
+
+                rideState.value.timeOfTrip?.let { time ->
+                    Text(
+                        text = "@$time",
+                        modifier = Modifier.align(CenterVertically),
+                        style = MaterialTheme.typography.body1
+                    )
+                }
+            }
+
+            Spacer(modifier = Modifier.height(20.dp))
+
+            Text(
+                "NUMBER OF TRAVELERS",
+                modifier = Modifier.wrapContentWidth(),
+                style = MaterialTheme.typography.subtitle2
+            )
+
+            Spacer(modifier = Modifier.height(5.dp))
+
+            Text(
+                String.format(
+                    "%d to %d people",
+                    rideState.value.minTravelers, rideState.value.maxTravelers
+                ),
+                style = MaterialTheme.typography.body1
+            )
+
+            Spacer(modifier = Modifier.height(30.dp))
+
+            Text(
+                "DETAILS",
+                modifier = Modifier.wrapContentWidth(),
+                style = MaterialTheme.typography.subtitle2
+            )
+
+            Spacer(modifier = Modifier.height(5.dp))
+
+            Text(
+                text = rideState.value.description ?: "None",
+                modifier = Modifier.wrapContentWidth(),
+                style = MaterialTheme.typography.body1
+            )
+
+            Spacer(modifier = Modifier.height(50.dp))
+
+            /*
+        Button(
+            onClick = { TODO },
+            modifier = Modifier
+                .height(50.dp)
+                .clip(RoundedCornerShape(100.dp))
+                .fillMaxWidth(),
+            colors = ButtonDefaults.buttonColors(
+                backgroundColor = DarkGreen,
+                contentColor = Color.White
+            )
+        ) {
+            Text(
+                stringResource(R.string.post_trip),
+                style = MaterialTheme.typography.body1.copy(Color.White)
+            )
+        }
+        */
+        }
+    }
+}
+
+@Composable
+fun ProfileMiniCard(
+    user: User
+) {
+    Row(
+        modifier = Modifier.fillMaxWidth()
+    ) {
+
+        Image(
+            // painter = rememberAsyncImagePainter(user.profilePicUrl),
+            painter = painterResource(R.drawable.ic_profile_placeholder),
+            contentDescription = null,
+            modifier = Modifier
+                .size(41.dp)
+                .clip(CircleShape)
+        )
```

Comment:
Can you add height(IntrinsicSize.Min) to the row modifier and replace size(41.dp) with fillMaxHeight in the image and see what that gets you? I'm trying to see if that aligns the image with the Column

Reply:
If this doesn't work, just add an align(Alignment.CenterVertically) to the modifier of the Image